### PR TITLE
Test CI for https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/300

### PR DESCRIPTION
**Pull request by pablogs9**
Close when pass

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/321/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/341/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/322/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/342/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@ endif()
 
 project(microxrcedds VERSION "2.1.0")
 
-set(_client_tag v2.1.0)
+set(_client_tag fix/fragmentation)
 set(_client_version 2.1.0)
 set(_agent_tag v2.1.0)
```
Comment (Acuadros95) [inline CMakeLists.txt:34]:
```suggestion
set(_agent_tag develop)
```
This should include latest version of the agent

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/323/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/343/)

**Pull request closed without merging**